README.Rmd

```diff
@@ -29,7 +29,7 @@ The goal of blazr is to provide R with a blazingly fast interface to multi-threa
 
 Install the development version of `blazr` from GitHub with:
 
-``` r
+```{r}
```

Comment:
I think this is not actually desirable. That code chunk was only meant to be a demo to users on how they _might_ want to install the package, not necessarily meant to run every time we knit the README.


I don't think we want the package to re-install from GitHub every-time we re-knit the README locally. (Consider if we are knitting the `README.Rmd` on a feature branch, it would be impossible/ very difficult to see those changes, as `pak` would always re-install from `main` on GitHub)

Reply:
I see where you're coming from. I'll revert this and add the package through the dependencies job.